.github/workflows/update-bridge.yml

```diff
@@ -93,6 +86,13 @@ jobs:
         pythonversion:
         - "3.7"
 "on":
-  repository_dispatch:
-    types:
-    - update-bridge
+  workflow_dispatch:
```

Comment:
this must also be a repostory_dispatch as we will be calling this remotely from the other repo

Reply:
I plan to use https://github.com/marketplace/actions/workflow-dispatch or to run `gh workflow run` to trigger from another repo, so if that's acceptable.

Reply:
I personally still don't see why we are insistent on changing this workflow but 🤷